images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -362,6 +365,15 @@ _ocp_enabled_overrides:
       config:
         prometheus:
           url: "https://prometheus-k8s.openshift-monitoring.svc:9091/"
+  presto:
```

Comment:
nit: can you add a newline between the reporting-operator prometheus URL section and the following presto section?

Reply:
Fixed in acfb7a89